app/src/main/java/org/jellyfin/androidtv/playback/overlay/actions/RecordAction.java

```diff
@@ -0,0 +1,23 @@
+package org.jellyfin.androidtv.playback.overlay.actions;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+
+import org.jellyfin.androidtv.R;
+import org.jellyfin.androidtv.playback.overlay.CustomPlaybackTransportControlGlue;
+
+public class RecordAction extends CustomAction {
```

Comment:
(addition to the previous suggestion for the action classes) adding a second icon parameter to the (to-be-made) constructor could trigger the active/inactive behavior. When the second icon is null it will always use the first one.

Reply:
Good point, but as you can see, I check which type of action is clicked in the onCustomActionClicked method in the CustomPlaybackTransportControlGlue class. But maybe I can instead use some sort of enum (eg. ACTION_SUBTITLES) which I pass in the IconAction object on initialization which I can later check against.